docs/finops/README.md

```diff
@@ -0,0 +1,46 @@
+---
+title: FinOps
+description: DevOps-like role in a collaborative approach to optimize cloud costs, ensuring financial accountability, efficiency, and resource management.
+sidebar_position: 9
+---
+
+# FinOps
+
+Understand financial principles relevant to cloud computing, like budgeting, forecasting, cost allocation, and billing.
```

Comment:
It's better to quote [Gartner's definition of the FinOps](https://www.gartner.com/en/information-technology/glossary/augmented-finops) here. So it's clear what it is that people don't know about it.